src/main/resources/static/openapi/SupportAdditionalNeedsAPI.yml

```diff
@@ -1473,6 +1492,25 @@ components:
           description: |
             An ISO-8601 date representing date that the plan creation is due.
           example: '2023-11-19'
+        planCompletedDate:
+          type: string
+          format: date
+          description: |
+            An ISO-8601 date representing date that the plan creation was created.
```

Comment:
How about we be really explicit about the intent/use of this field? Something like:

> If the status of this Plan Creation Schedule is COMPLETED, this field is an ISO-8601 date representing date that the Education Support Plan was created.
This field will only have a value when the status of the Plan Creation Schedule is COMPLETED, and reflects the date the Education Support Plan was created (rather than the Plan Creation Schedule)